src/oge/reference_tables/epa_eia_crosswalk_manual.csv

```diff
@@ -157,3 +157,37 @@ plant_id_epa,emissions_unit_id_epa,plant_id_eia,generator_id,start_year,end_year
 1571,SMECO,65285,GT1,2021,2022,This generator switches plant IDs for two years then switches back to its original plant ID
 1571,1,65285,ST1,2021,,This generator switches plant IDs the year it retires (2021)
 1571,2,65285,ST2,2021,,This generator switches plant IDs the year it retires (2021)
+3,8,3,A3C1,2023,,This generator (CEMS unit) had no CEMS unit (generator) assigned

```

Comment:
We should leave the start_year row empty since this mapping would apply in all years

Reply:
Removed `start_year` for all added mapping